stix2/observables.py

```diff
@@ -750,6 +766,16 @@ class _Custom(cls, _Observable):
             _properties = {
                 'type': TypeProperty(_type),
             }
+            # Check properties ending in "_ref/s" are ObjectReferenceProperties
+            for prop_name, prop in properties.items():
+                if prop_name.endswith('_ref') and prop.__class__.__name__ != 'ObjectReferenceProperty':
+                    raise ValueError("'%s' is named like an object reference property but "
+                                     "is not an ObjectReferenceProperty." % prop_name)
+                elif (prop_name.endswith('_refs') and (prop.__class__.__name__ != 'ListProperty'
+                                                       or prop.contained.__class__.__name__ != 'ObjectReferenceProperty')):
+                    raise ValueError("'%s' is named like an object reference property but "
```

Comment:
Do we want this to be different from the error message above ("is named like an object reference *list* property")?